client/verta/verta/registry/entities/_modelversion.py

```diff
@@ -865,8 +856,8 @@ def _upload_artifact(
 
     def _get_artifact(self, key, artifact_type=0):
         # check to see if key exists
-        self._get_artifact_msg(key)
-
+        artifact_msg = self._get_artifact_msg(key)
+        self._validate_artifact_uploaded(artifact_msg)
```

Comment:
Since `artifact_msg` goes unused, I'd request not binding it to a variable at all, unless it seems more readable this way.

```suggestion
        self._validate_artifact_uploaded(self._get_artifact_msg(key))
```